.spelling

```diff
@@ -285,6 +285,11 @@ v0
 ipfsdaemon
 py3.6
 async
+gnosis
+abci
+metaclass
+abstractroundbehaviour's
```

Comment:
that's not a word

Reply:
I had an oracle agent in the open-aea repo for some testing purposes, and it had `AbstractRoundBehaviour's` in the readme file. I don't wanna fix this right now since the CI is clogged. I'll push once all the tests pass.